cmd/access/access_node_builder.go

```diff
@@ -178,57 +185,57 @@ func (builder *FlowAccessNodeBuilder) parseFlags() {
 // The factory function is later passed into the initMiddleware function to eventually instantiate the p2p.LibP2PNode instance
 func (builder *FlowAccessNodeBuilder) initLibP2PFactory(ctx context.Context,
 	nodeID flow.Identifier,
-	networkMetrics module.NetworkMetrics,
 	networkKey crypto.PrivateKey) (p2p.LibP2PFactoryFunc, error) {
 
-	// setup the Ping provider to return the software version and the sealed block height
-	pingProvider := p2p.PingInfoProviderImpl{
```

Comment:
`PingProvider` is used to ping all Flow nodes and query their software version and chain height. For the unstaked side of things, we probably don't need it in the first version.